packages/node_modules/@webex/react-component-space-item/src/momentum.scss

```diff
@@ -0,0 +1,9 @@
+$brand-font-folder: "~@momentum-ui/core/fonts";
+$icon-font-path: "~@momentum-ui/icons/fonts";
+
+@import '~@momentum-ui/core/scss/momentum-ui-components';
+
+// Workaround for missing style
+.md-decrypting {
```

Comment:
This style is not currently included in the momentum-ui-components package. They are adding it later.